spec/support/view_component.rb

```diff
@@ -14,5 +15,6 @@ module ViewComponent
   config.include Spec::Support::ViewComponent, type: :component
   config.include ViewComponent::TestHelpers, type: :component
   config.include ViewComponent::SystemTestHelpers, type: :component
+  config.include Capybara::DSL, type: :component
```

Comment:
This isn't necessary anymore since I'm not using `visit` in the `ComponentSpec` anymore